mmhuman3d/core/cameras/cameras.py

```diff
@@ -756,6 +759,17 @@ def get_default_projection_matrix(cls, **args) -> torch.Tensor:
             principal_point=principal_point,
             orthographic=False)
 
+    def get_ndc_camera_transform(self, **kwargs) -> Transform3d:
```

Comment:
> `get_ndc_camera_transform` and `transform_points_screen` has been duplicated several times. Can we move these two functions to `NewAttributeCameras` that inherits PyTorch3D's [`CameraBase`](https://pytorch3d.readthedocs.io/en/latest/modules/renderer/cameras.html). Hence, as all our cameras inherits `NewAttributeCameras`, they will also inherit `get_ndc_camera_transform` and `transform_points_screen`.

I did as you said initially, but this triggers some errors. Actually, I use multi inherit in `cameras`, E.g., `class PerspectiveCameras(cameras.PerspectiveCameras, NewAttributeCameras)`, in the `__mro__` chain, `PerspectiveCameras` will inherit `pytorch3d PerspectiveCameras` first, then `NewAttributeCameras`. This is mainly because the need of some specific functions. (For example, I want our `PerspectiveCameras` to directly initialize from `NewAttributeCameras` and avoid inheriting the `pytorch3d PerspectiveCameras`, thus the multi inherit should be like this). While in function `get_ndc_camera_transform` and `transform_points_screen`, they will meet `unexpected keyword error` because in some APIs it will be feed with multi `cameras` keyword if you do not pop it. If I write  `get_ndc_camera_transform` and `transform_points_screen` in `NewAttributeCameras`, as the mro queue, it will call the function with same name in `pytorch3d PerspectiveCameras` first, then `NewAttributeCameras`. Or I should mannually super(cameras.PerspectiveCameras, self) for each function. So I think I should just write directly for each class.